src/storage/test/RebuildIndexTest.cpp

```diff
@@ -75,7 +75,8 @@ TEST_F(RebuildIndexTest, RebuildTagIndexCheckALLData) {
     parameter.set_space_id(1);
     std::vector<PartitionID> parts = {1, 2, 3, 4, 5, 6};
     parameter.set_parts(parts);
-    parameter.task_specfic_paras.emplace_back("4,5");
+    parameter.task_specfic_paras.emplace_back("4");
```

Comment:
IF we will add more argument about rebuild how to determine the index id bounds ？

Reply:
Can you tell me what parameters will be supported?  I think every parameter in the vector means the same meaning. so the user  use it is easy. and it no need to tell user the firstly parameter mean what, the second mean what.

Reply:
There are hidden dangers here